src/test/cli/radosgw-admin/help.t

```diff
@@ -42,6 +42,7 @@
     object stat                stat an object for its metadata
     object unlink              unlink object from bucket index
     object rewrite             rewrite the specified object
+    object reindex             reindex the object(s) indicated by --bucket and either --object or --objects-file
```

Comment:
the --objects-file and --infile changes to radosgw-admin usage will also need to be reflected here